cmd/influx_tools/export/README.md

```diff
@@ -73,4 +73,99 @@ Seq #           ID              Start                           End
+The range refers to the output shard sequence numbers (the first column of the second table in the plan output).
+
+Once happy with the plan, run without `-print-only` (make sure to pipe to a file!):
+
+```sh
+$ ./influx_tools export -config /root/.influxdb/influxdb.conf -database collapsedb -duration 4032h -format binary -conflict-path /conflicts.tar.gz -range '0-0' > export.binary
+Source data from: 2020-05-25 00:00:00 +0000 UTC -> 2021-07-19 00:00:00 +0000 UTC
+
+Converting source from 60 shard group(s) to 1 shard groups:
+
+Seq #           ID              Start                           End
+0               9               2020-05-25 00:00:00 +0000 UTC   2020-06-01 00:00:00 +0000 UTC
+1               10              2020-06-01 00:00:00 +0000 UTC   2020-06-08 00:00:00 +0000 UTC
+2               11              2020-06-08 00:00:00 +0000 UTC   2020-06-15 00:00:00 +0000 UTC
+3               12              2020-06-15 00:00:00 +0000 UTC   2020-06-22 00:00:00 +0000 UTC
+…
+```
+
+The import step is next, which requires a period of data unavailability.
+
+1) For safety, take a full database backup - see https://docs.influxdata.com/enterprise_influxdb/v1.9/administration/backup-and-restore/
+2) With all nodes running, delete *all* the shards that would overlap with the newly created shard(s):
+    1) The output from the export plan (the ‘ID’ column) gives the shard group id’s
+    2) The influxql `SHOW SHARDS` command gives the shard id’s (`id` column) for the shard group id’s (`shard_group` column)
+    3) The influxql `DROP SHARDS` command is able to drop the old shards
+3) Take one node down, and use `influx_tools import` on that node to import the data, similar to how `influx_tools export`
+   was used on the running backup . Note `-shard-duration` is actually the shard group duration, and `duration`
+   should match the existing time to expiry (default is infinite). If necessary, temporarily update the retention policy
+   to a shard duration matching the one chosen during export (otherwise you will get an error `import: retention policy
+   autogen already exists with different parameters`). E.g. `alter retention policy autogen on collapsedb shard duration 4032h`.
+   **If this is necessary, ensure there are no other writes to the database that would create a new shard during this time.**
+
+```sh
+$ ./influx_tools import -config /root/.influxdb/influxdb.conf -database collapsedb -rp autogen -shard-duration 4032h < export.binary
+{"level":"info","ts":1626446664.3253446,"caller":"importer/command.go:64","msg":"import starting"}
+{"level":"info","ts":1626446664.3271663,"caller":"importer/importer.go:107","msg":"Starting shard group 2020-04-06 00:00:00 +0000 UTC-2020-09-21 00:00:00 +0000 UTC with 0 existing shard groups"}
+```
+
+4) Bring the node back up, and use `influxd-ctl copy-shard` to copy the new shard(s) to the other cluster node:
```

Comment:
Providing the recommended method for bringing the node up or down would add clarity

Reply:
That unfortunately depends on the customer's setup, so I can't give a good recommendation here.